Statistics/AlertHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Statistics
+{
+    public class AlertHandler
+    {   
+        List<IAlert> Alerters = new List<IAlert>();
+         public double threshold;
+
+
+        public AlertHandler(double maxThreshold, List<IAlert> alerters)
+        {
+            threshold = maxThreshold;
+            for (int i = 0; i < alerters.Count; i++)
+                Alerters.Add(alerters[i]);
+        }
+       
+        public void CheckStatisticsandAlert(List<float> numbers)
+        {
+            var statsComputer = new StatsComputer();
+            var max = statsComputer.CalculateStatistics(numbers).max;
+     
+          
+            foreach(IAlert alerter in Alerters)
+            { 
+                alerter.Alert(threshold, max);
```

Comment:
A reader would normally expect a `.Alert` call to simply 'alert'. However, here the alert has a comparison inside, so may not alert as well. Separate the concerns - keep comparison and alerting separate.